<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightblue;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="loginTextEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginPushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalidUserLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 86, 87);</string>
    </property>
    <property name="text">
     <string>   Invalid User</string>
    </property>
   </widget>
   <widget class="QLabel" name="enterUserLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;</string>
    </property>
    <property name="text">
     <string> Enter a User Name Here:</string>
    </property>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 25px;
color: black;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="recentPostsLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 10px;
color: black;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendsTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>256</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToOwnProfile">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 15px;
color: black;</string>
    </property>
    <property name="text">
     <string>Back to My Profile</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="suggestTable">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;
font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPostButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Add Post</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="postTextEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="postButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Post</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addReactionButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Add Reaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seeReactionsButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>See Reactions</string>
    </property>
   </widget>
   <widget class="QLabel" name="postsLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>  Posts: </string>
    </property>
   </widget>
   <widget class="QLabel" name="choosePostLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>Choose Post:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="reactionsLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 12px;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reactionsDoneButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: white;</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button22">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button33">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button44">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button55">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseReactionLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string>Choose Reaction:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likeButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 50px;
color: black;</string>
    </property>
    <property name="text">
     <string>üëç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="heartButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 50px;
color: black;</string>
    </property>
    <property name="text">
     <string>‚ù§Ô∏è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="laughButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font-size: 50px;
color: black;</string>
    </property>
    <property name="text">
     <string>üòÇ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

